setup.py

```diff
@@ -17,22 +17,54 @@
 import sys
 import subprocess
 import shutil
+import json
 from pathlib import Path
 from setuptools import setup, find_packages
 
+def envar():
+    backends = {"CUDA", "HIP", "OPENMP", "THREADS", "SERIAL"}
+    archs = {"VEGA900","VEGA906","VEGA908","VEGA90A"}
```

Comment:
`archs`, in this context, seems to refer only to AMD GPUs. What would happen with it in a CPU backend case?